public/app/features/datasources/components/DataSourceCategories.tsx

```diff
@@ -15,6 +19,14 @@ export type Props = {
 };
 
 export function DataSourceCategories({ categories, onClickDataSourceType }: Props) {
+  const destinationLinks = {
+    [DestinationPage.dataSources]: ROUTES.DataSources,
+    [DestinationPage.connectData]: ROUTES.ConnectData,
+  };
```

Comment:
As `destinationLinks[DestinationPage.dataSources]` is not used, maybe we can remove the `destinationLinks` object and just inline the route, e.g. like this:

```javascript
const moreDataSourcesLink = config.featureToggles.dataConnectionsConsole
    ? `${ROUTES.ConnectData}?cat=data-source`
    : '/plugins?filterBy=all&filterByType=datasource&utm_source=grafana_add_ds';
```

Reply:
Yep, thanks - corrected https://github.com/grafana/grafana/pull/63517/commits/ca8e1d6fd98106da8e28a4bc8ea41283ecbdf713

Reply:
Please take a look again @leventebalogh